updateQDKVersion.ps1

```diff
@@ -0,0 +1,61 @@
+$ver = $args[0]
+Write-Host $ver
+
+$csString = 'PackageReference Include=\"Microsoft\.Quantum\.[a-zA-Z\.]+\" Version=\"(?<oldVersion>[0-9|\.]+)\"'
+$csFiles = (Select-String -Path "**\*.csproj" -pattern "Microsoft.Quantum" | Select-Object -Unique Path)
```

Comment:
I think this mask for the Path doesn't catch the files in `tutorials` subfolder, since they are nested one level deeper than the others. Do you know what PowerShell mask corresponds to "a file arbitrarily deep in the tree"? (Same change will need to be applied to the `.ipynb` pattern)

Reply:
One way to explicitly get arbitrarily deep files would be with `Get-ChildItem -Recurse`:
```powershell
Get-ChildItem -Recurse *.csproj `
    | ForEach-Object { Select-String -Path $_ -Pattern "Microsoft.Quantum" } `
    | Select-Object -Unique Path
```